package.json

```diff
@@ -94,5 +107,10 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "taskr": {
```

Comment:
Haven't seen this before, happy to give it a go. 

Just wondering what your thoughts on why it's good to use?

Reply:
🔥️ removed `taskr` went back to `@babel/cli`